tools/makeboards.py

```diff
@@ -414,6 +414,9 @@ def MakeBoardJSON(name, vendor_name, product_name, vid, pid, pwr, boarddefine, f
 MakeBoard("pimoroni_plasma2040", "Pimoroni", "Plasma2040", "0x2e8a", "0x100a", 500, "PIMORONI_PLASMA2040", 2, "boot2_w25q080_2_padded_checksum")
 MakeBoard("pimoroni_tiny2040", "Pimoroni", "Tiny2040", "0x2e8a", "0x100a", 500, "PIMORONI_TINY2040", 2, "boot2_w25q64jv_4_padded_checksum")
 
+# RAKwireless
+MakeBoard("rakwireless_rak11300", "RAKwireless", "RAK11300", "0x2e8a", "0x00c0", 500, "RAKWIRELESS_RAK11300", 2, "boot2_w25q080_2_padded_checksum")
```

Comment:
If you add `, None, "https://store.rakwireless.com/products/wisduo-lpwan-module-rak11300` in the arugment list, then that URL can show up in the board definition too, instead of just the generic Pico one.

Reply:
Thanks, I did that, also more closely specified the used flash chip as I took one of this modules apart under a microscope :)